.github/ISSUE_TEMPLATE/issue_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Issue Report
+about: Report an issue to help us improve
+title: '[ISSUE] '
+---
+
+**Article and Module Links**
+A link to the affected article and the affected module. The link to the module is the one in the "over on GitHub" standard phrase.
```

Comment:
We can write here that "You can find the link to the module in the conclusion section in the "on Github" standard phase."